FlowPluginsTs/CommunityFlowPlugins/ffmpegCommand/ffmpegCommandEnsureAudioStream/1.0.0/index.ts

```diff
@@ -245,18 +300,27 @@ const attemptMakeStream = ({
   const streamCopy: IffmpegCommandStream = JSON.parse(JSON.stringify(streamWithHighestChannel));
   streamCopy.removed = false;
   streamCopy.index = streams.length;
+  if (targetChannels > highestChannelCount && upmixingFilter !== '') {
+    streamCopy.outputArgs.push('-filter_complex', `[${streamCopy.mapArgs[1]}]${upmixingFilter}[a_{outputIndex}]`);
+    streamCopy.outputArgs.push('-map', '[a_{outputIndex}]');
+    streamCopy.mapArgs = [];
```

Comment:
need to remove the mapArgs here because otherwise the source will be copied twice